core/iree-requirements.txt

```diff
@@ -0,0 +1,2 @@
+iree-compiler==20240129.785
```

Comment:
Any thoughts on updating this to latest iree-compiler/iree-runtime release?

Reply:
I bumped it last week. We need to do it regularly.

Reply:
(but in dedicated patches, not incidentally for no reason)

Reply:
Sounds good!